packages/hooks/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@leafygreen-ui/hooks",
+  "version": "0.9.0",
```

Comment:
I think having this as 1.0.0 is fine, no?

Reply:
lerna version bumps to 1.0.0 when we release, so @DesignerDave was in favor of starting off at 0.9.0, so the first release is 1.0.0 instead of 1.0.1

Reply:
Ah, didn't know that. Neat! TIL.

Reply:
Learnings from using Lerna publish hah